tests/test_new_talk_voting.py

```diff
@@ -143,7 +302,7 @@ def test_dont_publish_talks_without_speaker_details(db, user_client, mocker):
     assert talk2.title in response.content.decode("utf8")
 
 
-def test_dont_publish_talks_without_speaker_details1(db, user_client, mocker):
+def test_dont_publish_talks_without_speaker_details1(db, user_client):
```

Comment:
why the `1`?

Reply:
Must have come from an earlier MR; looks like there's two tests with near identical names, one testing the view, the other testing the queryset. Will update names accordingly.